.github/workflows/test-external-providers.yml

```diff
@@ -5,6 +5,10 @@ on:
     branches: [ main ]
   pull_request:
     branches: [ main ]
+    paths-ignore:
```

Comment:
Wondering if we should just do like unit and integration tests and use:

```
    paths:
      - 'distributions/**'
      - 'llama_stack/**'
      - 'tests/integration/**'
      - 'uv.lock'
      - 'pyproject.toml'
      - 'requirements.txt'
   ```

Reply:
I actually thought about that too and I figured I'd do the ignore but I can adjust it to this for sure!